### DISCUSSION TITLE
Option to download model information/metadata as well

### DISCUSSION THREAD

**Discussion by apampuch** [civitai/civitai#445] [Ideas]
Quite often I download a model and forget what the hell it is. While I'd like to think that civtai will be around forever, I don't think that's going to be the case long-term. Therefore, I'd like an option to download the model info as well, even if it's only just a text readme with the same name minus the extension.

**Comment by malcolmamal** [comment]
I would extend this request with the following:

* add a "download metadata/additional data" that would prepare a zip for you containing:
    - model's text description with triggers
    - all the official image samples (in original form, so with the metadata)
